buildenv/jenkins/README.md

```diff
@@ -245,25 +245,25 @@ The [Running AdoptOpenJDK Tests](https://github.com/AdoptOpenJDK/openjdk-tests/b
 
 #### Infrastructure
 
-Infrastructure pipelines are available [**here**](https://ci.eclipse.org/openj9/view/Infrastructure/)
+Infrastructure pipelines are available [**here**](https://openj9-jenkins.osuosl.org/view/Infrastructure/)
 
 - Mirror-OMR-to-OpenJ9-OMR
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Mirror-OMR-to-OpenJ9-OMR)](https://ci.eclipse.org/openj9/job/Mirror-OMR-to-OpenJ9-OMR)
+    - [![Build Status](https://openj9-jenkins.osuosl.org/buildStatus/icon?job=Mirror-OMR-to-OpenJ9-OMR)](https://openj9-jenkins.osuosl.org/job/Mirror-OMR-to-OpenJ9-OMR)
     - Description:
         - Mirrors [eclipse/omr/master](https://github.com/eclipse/omr/tree/master) to [eclipse-openj9/openj9-omr/master](https://github.com/eclipse-openj9/openj9-omr/tree/master)
         - Triggers `Pipeline-OMR-Acceptance` when there is new content
     - Trigger:
         - Build periodically, 15 minutes
 
 - Promote_OMR
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Promote_OMR)](https://ci.eclipse.org/openj9/job/Promote_OMR)
+    - [![Build Status](https://openj9-jenkins.osuosl.org/buildStatus/icon?job=Promote_OMR)](https://openj9-jenkins.osuosl.org/job/Promote_OMR)
     - Description:
         - Promotes eclipse-openj9/openj9-omr branch master to branch openj9
     - Trigger:
         - Last step of `Pipeline-OMR-Acceptance`
 
 - Mirror-OpenJ9-Website-to-Eclipse
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Mirror-OpenJ9-Website-to-Eclipse)](https://ci.eclipse.org/openj9/job/Mirror-OpenJ9-Website-to-Eclipse)
+    - [![Build Status](https://openj9-jenkins.osuosl.org/buildStatus/icon?job=Mirror-OpenJ9-Website-to-Eclipse)](https://openj9-jenkins.osuosl.org/job/Mirror-OpenJ9-Website-to-Eclipse)
```

Comment:
This link doesn't work

Reply:
```suggestion
    - [![Build Status](https://openj9-jenkins.osuosl.org/buildStatus/icon?job=Deploy-OpenJ9-Website-to-Eclipse)](https://openj9-jenkins.osuosl.org/job/Deploy-OpenJ9-Website-to-Eclipsee)
```

Reply:
Will fix in a rebase as there's a typo in the URL.